.github/actions/linux-web-init-and-check/action.yml

```diff
@@ -0,0 +1,71 @@
+name: "Web Build Initialize and Check on Linux"
+description: "Initializes and checks the ONNX Runtime Web build on Linux."
+runs:
+  using: "composite"
+  steps:
+    - name: Setup Node.js
+      uses: actions/setup-node@v3
+      with:
+        node-version: "22.x"
+
+    - name: npm ci /js/
+      run: npm ci
+      shell: bash
+      working-directory: ${{ github.workspace }}/js
+
+    - name: npm ci /js/common/
+      run: npm ci
+      shell: bash
+      working-directory: ${{ github.workspace }}/js/common
+
+    - name: run onnxruntime-common tests
+      run: npm test
+      shell: bash
+      working-directory: ${{ github.workspace }}/js/common
+
+    - name: run onnxruntime-common tests (enable Float16Array)
+      run: npm run test:f16
+      shell: bash
+      working-directory: ${{ github.workspace }}/js/common
+
+    - name: npm ci /js/web/
+      run: npm ci
+      shell: bash
+      working-directory: ${{ github.workspace }}/js/web
+
+    - name: run TypeScript type check in /js/web/
+      run: npm run prebuild
+      shell: bash
+      working-directory: ${{ github.workspace }}/js/web
+
+    - name: run ESLint
+      run: npm run lint
+      shell: bash
+      working-directory: ${{ github.workspace }}/js
+
+    - name: Clang-format
```

Comment:
Does this step rely on clang? What if the clang version gets changed?

Reply:
It is no longer clang-format. the formatter is prettier now. let me fix this name for less confusing.